app/Console/Commands/GenerateRobots.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class GenerateRobots extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'robots:generate';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate the robots.';
```

Comment:
Given that robots could have different meanings, it might make sense to say robots.txt file

```suggestion
    protected $description = 'Generate the robots.txt file.';
```